samples/python/pytorch_transformer/ort_train.py

```diff
@@ -0,0 +1,85 @@
+import argparse
```

Comment:
From the diff these seem like totally new files and not updates. :-)
Is that how it is supposed to be?